website/blog/unusual-builds-w-bytes.md

```diff
@@ -0,0 +1,228 @@
+---
+slug: unusual-builds-w-bytes
+title: "Unusual Builds with Bytes"
+description: "Fixing a download bug in Bazel Build without Bytes (BwoB)"
+authors: son
+date: 2025-03-10:12:00:00
+image: /img/blog/troubleshooting.png
```

Comment:
TODO: replace this image. (currently re-using the one from the bisect blog post)